setup.cfg

```diff
@@ -20,6 +20,7 @@ classifiers =
     Framework :: Django
     Framework :: Django :: 4.2
     Framework :: Django :: 5.0
+    Framework :: Django :: 5.1
```

Comment:
drop line 14 and 15 (`Programming Language :: Python :: 3.8`, `Programming Language :: Python :: 3.9`)